src/Nfield.Quota/QuotaFrameValidator.cs

```diff
@@ -17,10 +17,14 @@ public QuotaFrameValidator()
             RuleFor(qf => qf.VariableDefinitions)
                 .Must(HaveUniqueIds)
                 .WithMessage("Quota frame definitions contain a duplicate id. Duplicate id: '{DuplicateValue}'")
-                .Must(HaveUniqueNames)
+                .Must(HaveUniqueVariableNames)
                 .WithMessage("Quota frame definitions contain a duplicate name. Duplicate name: '{DuplicateValue}'")
                 .Must(HaveVariablesWithAtLeastOneLevel)
-                .WithMessage("Quota frame definitions has variables with no levels. Affected variable definition id: '{VariableDefinitionId}'");
+                .WithMessage("Quota frame definitions has variables with no levels. Affected variable definition id: '{VariableDefinitionId}'")
```

Comment:
Maybe if we steer customers towards using names, we should also use them in our messages